.env

```
@@ -0,0 +1,13 @@
+  type= "service_account"
+  project_id= "express-sheets-demo"
+  private_key_id= "sample-private-key-id"
+  private_key= "sample-private-key"
+  client_email= "sample-client-email"
+  client_id= "sample-client-id"
+  auth_uri= "sample-auth-uri"
+  token_uri= "sample-token-uri"
+  auth_provider_x509_cert_url= "sample-auth-provider-cert"
+  client_x509_cert_url = "sample-client-cert"
+  sheetId= "sample-sheetid"
```

inconsistent spacing
```suggestion
  sheetId = "sample-sheetid"
```

---

Done